src/components/PlantCardPreview/PlantCardPreview.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Text, Image } from 'react-native';
+import { useTranslation } from 'react-i18next';
+import { useStyles } from 'react-native-unistyles';
+import { styleSheet } from './PlantCardPreview.style';
+import { View } from '../Themed';
+import { PlantCardInfo } from '../PlantCardInfo/PlantCardInfo';
+import { Plant } from '@src/api/model/plant';
```

Comment:
PLease consider using `FastImage` instead, since it is a better performing solution